### PR TITLE
Frontend: Support tracing with Faro

### LINKED ISSUE #90116: Implement Faro web tracing
Implement Faro's web tracing support in Grafana using `@grafana/faro-web-tracing` or `@opentelemetry/instrumentation-fetch`.

See:
 - https://github.com/grafana/grafana/pull/89776
 - https://grafana.com/docs/grafana-cloud/monitor-applications/frontend-observability/instrument/opentelemetry-js/
 - https://grafana.com/docs/grafana-cloud/monitor-applications/frontend-observability/instrument/tracing-instrumentation/
 - https://raintank-corp.slack.com/archives/C01T8C5CNTE/p1719415687646959

### PR DESCRIPTION
**Pull request by hairyhenderson**
**What is this feature?**

Allows enabling frontend tracing with Faro.

(this is still a work in progress - I haven't been able to get it working locally yet)

Related to https://github.com/grafana/grafana/issues/88930 too apparently!

Closes #90116

**Who is this feature for?**

Operators

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -1494,6 +1494,10 @@ Turn on console instrumentation. Only affects Grafana Javascript Agent
 
 Turn on webvitals instrumentation. Only affects Grafana Javascript Agent
 
+### instrumentations_tracing_enabled
+
+Turn on tracing instrumentation. Only affects Grafana Javascript Agent
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1499]:
```suggestion
Turns on tracing instrumentation. Only affects Grafana Javascript Agent.
```

**Review by chri2547 — COMMENTED** [review]
One suggestion. Docs approved.

**Comment by tskarhed** [comment]
Hmm, getting SASS errors, they seem to appear in main 🤔

**Comment by tskarhed** [comment]
Closing this in favor of #91237, because the CI is very weird here. It is essentially the same PR.

**Pull request closed without merging**